src/LabelSelector/LabelChipGroup/LabelChipGroup.tsx

```diff
@@ -88,6 +89,20 @@ export default function LabelChipGroup({ chips }: LabelChipGroupProps) {
     }
   }, [chips, moreItemsRef, parentRef]);
 
+  // custom onClick event handler for label chips shown in the popper to close the popper on click
+  const handleOverflowingChipClick = (
+    event: React.MouseEvent<HTMLDivElement>,
+    onClick: LabelChipProps["onClick"]
```

Comment:
shouldn't we also check whether the LabelChip is clickable? Right now if I click an unclickable LabelChip, it will close the popover regardless. Or is there a reason we want it this way?

Reply:
How are you clicking an unclickable label?

Reply:
I set the `clickable` prop of the chips in storybook to false (tried it with just deleting the prop, works the same as expected), then when I click on the label, it still closes the LabelChipGroup, see the screen recording attached.

https://github.com/user-attachments/assets/cea08dd7-2f31-4f86-8a51-2867e244fb6d

Reply:
This sounds like an existing issue with the LabelChip component and/or the underlying MUI chip component. Seems strange that onClick is getting fired when clickable is set to false.

Reply:
Indeed it's actually the default behaviour of the MUI chip component it seems. https://mui.com/material-ui/api/chip/

![image](https://github.com/user-attachments/assets/8160ac36-2583-4a9d-8b1e-25ecc0dc341c)

I can update our usage of the MUI Chip component to change this behaviour.

Reply:
I was just texting the same out 😄 Sounds good, thanks 👍

Reply:
Seems like we also have an existing (badly written) test that is trying to test for this. It currently passes because the test is terrible.

![image](https://github.com/user-attachments/assets/489579c8-a8d5-4e70-a6d4-379d6aea1448)